sql/core/src/main/scala/org/apache/spark/sql/catalyst/analysis/ResolveSessionCatalog.scala

```diff
@@ -373,24 +369,21 @@ class ResolveSessionCatalog(
 
     case d @ DropNamespace(SessionCatalogAndNamespace(_, ns), _, _) =>
       if (ns.length != 1) {
-        throw new AnalysisException(
-          s"The database name is not valid: ${ns.quoted}")
+        throw new AnalysisException( s"The database name is not valid: ${ns.quoted}")
```

Comment:
This new statement also has a wrong space, `( s"`. Please revert this irrelevant change.